java/examples/binary/README.md

```diff
@@ -0,0 +1,14 @@
+# s2i-java-binary-example
+
+This S2I example illustrates how to build a container image for a previously existing Java binary JAR. 
+In this example, one is copied from ../maven/target; but typically you instead would have it downloaded from somewhere.
```

Comment:
real link ?

Reply:
no in this case it doesn't really make much sense IMHO, because `../maven/target` won't be available on (GitHub) sources to click through, so I would expect this would be more confusing than helpful, if anyone ever really did click the link.

Reply:
ok, got it.